src/main/java/codeu/controller/ActivityServlet.java

```diff
@@ -1,27 +1,83 @@
 
-
 package codeu.controller;
 
+import codeu.model.data.Conversation;
+import codeu.model.data.Message;
 import java.io.IOException;
 
-import codeu.model.data.User;
+import codeu.model.store.basic.ConversationStore;
+import codeu.model.store.basic.MessageStore;
 import codeu.model.store.basic.UserStore;
+import java.util.List;
+import java.util.UUID;
 
 import javax.servlet.ServletException;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+/*Servlet class responsible for the activity feed page*/
 public class ActivityServlet extends HttpServlet {
+        
+    /*Store class that gives access to users*/
+	private UserStore userStore;
+    /*Store class that gives access to conversations*/
+	private ConversationStore conversationStore;
+	/** Store class that gives access to Messages. */
+	private MessageStore messageStore;
+
+
+	/**
+	* Set up state for handling conversation-related requests. This method is only called when
```

Comment:
nit:
Misalignment of comment.

The asterisk at the beginning of '* Setup' should align with the second asterisk from the first line '/**'